### PR TITLE
Update `README.md`

### PR DESCRIPTION
**Pull request by jpaillard**
- Add "about green" section presenting an overview of the model
 - detail dependencies
 - Document where examples are located 
 - Provide minimal snippet to integrate with braindecode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,18 +19,59 @@
...
+	dropout=0.5,		          # Dropout rate of 0.5 in FC layers
+	hidden_dim=[100],             # Use 100 units in the hidden layer
+	pool_layer=RealCovariance(),  # Compute covariance after wavelet transform
+	bi_out=[20],    		      # Use a BiMap layer outputing a 20x20 matrix
+	out_dim=1, 				      # Output dimension of 1, for regression
+)
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+EarlyStopping(monitor="valid_loss", patience=10, load_best=True)
+clf = EEGClassifier(
```
Comment (bruAristimunha) [inline README.md]:
```suggestion
clf = EEGRegressor(
```

Comment (jpaillard) [reply README.md]: thx !

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 # GREEN architecture
 ![CI](https://github.com/Roche/neuro-green/actions/workflows/lint_and_test.yaml/badge.svg)
 ---
+
+## About the architecture
+The model is a deep learning architecture designed for EEG data that combines wavelet transforms and Riemannian geometry. It is capable of learning from raw EEG data and can be applied to both classification and regression tasks. The model is composed of the following layers:
+It is based on the following layers:
```
Comment (dengemann) [inline README.md:7]: duplicated meaning / line.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+ - Convolution: Uses complex-valued Gabor wavelets with parameters that are learned during training.
```
Comment (dengemann) [inline README.md:9]: maybe specify which parameters, center frequency and width?

File: README.md [hunk 4]
```diff
@@ -9,18 +19,59 @@
...
+In addition, being pure PyTorch, the GREEN model can easily be integrated to [`braindecode`](https://braindecode.org/stable/index.html) routines. 
+
+```python
+import torch
+from braindecode import EEGRegressor
+from green.wavelet_layers import RealCovariance
+from green.research_code.pl_utils import get_green
+
+green_model = get_green(
+	n_freqs=5,                    # Learning 5 wavelets
```
Comment (dengemann) [inline README.md]: is this a 4-space indent? if not please do.

**Pull request merged**